mmdet/ops/nms/nms_wrapper.py

```diff
@@ -12,14 +12,17 @@ def nms(dets, thresh, device_id=None):
     if isinstance(dets, torch.Tensor):
         if dets.is_cuda:
             device_id = dets.get_device()
-        dets = dets.detach().cpu().numpy()
-    assert isinstance(dets, np.ndarray)
+        _dets = dets.detach().cpu().numpy()
+    else:
+        _dets = dets.copy()
```

Comment:
We don't need to copy the original array since it will not be modified in-place.

Reply:
Thanks!